openfisca_uk_data/datasets/frs/frs.py

```diff
@@ -296,18 +296,30 @@ def add_household_variables(frs: h5py.File, household: DataFrame):
 
     # Impute Council Tax
 
-    region = household.GVTREGNO.fillna("N/A")
-    band = household.CTBAND.fillna("N/A")
-    single_person = (household.ADULTH == 1).fillna("N/A")
-    CT_mean = household.groupby(
+    CT_valid = household.CTANNUAL > 0
```

Comment:
could you add a couple comments on this section?

Reply:
Added